File: Assignments/2_Operators and Expression/C#/Q2.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Enter principal amount (P): ");
        double principal = double.Parse(Console.ReadLine());
        Console.Write("Enter annual interest rate (r) in percentage: ");
        double annualRate = double.Parse(Console.ReadLine());
        Console.Write("Enter investment tenure (t) in years: ");
        int years = int.Parse(Console.ReadLine());

        double amount = principal * Math.Pow((1 + (annualRate / 100) / 12), 12 * years);
        Console.WriteLine($"Compound Interest after {years} years: {amount:F2}");
    }
}

File: Assignments/2_Operators and Expression/Javascript/Solutions/Q2.js
function calculateCompoundInterest(P, r, t) {
    const n = 12;
    const A = P * Math.pow(1 + (r / 100) / n, n * t);
    return A;
}

const P = 1000;
const r = 5;
const t = 10;
const amount = calculateCompoundInterest(P, r, t);
console.log(`Compound Interest after ${t} years: ${amount.toFixed(2)}`);
